### PR TITLE
Use usermod/groupmod to modify the user/group at runtime

### PR DESCRIPTION
**Pull request by robertbeal**
- Update to a newer and more specific version of python alpine (so the shadow package is available)
- Update the readme instructions

This gives people using the container another option for how they want to run it :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,12 +57,25 @@ docker run -d --name radicale \
...
+     -v ~/radicale/config:/config:ro \
+    tomsquest/docker-radicale
+```
 
 ## User/Group ID
 
 If you want another user to run the docker container and so "share" files with fixed permission between the host and the container, two options:
 1. Create a user on your host with ID `2999` (hardcoded in the built image): `useradd --uid 2999 radicale`
-1. Or build the image yourself and specify the user ID you want: `docker build -t radicale --build-arg=UID=5000 --build-arg=GID=5001 .` (see the [Building](#Building) section below)
+2. Specify `-e UID=123` and `-e GID=456` for user and group Id's. `--read-only` is not possible with this method as it modifes the filesystem at runtime.
```
Comment (tomsquest) [inline README.md]: typo ìn `modifes`

**Comment by tomsquest** [comment]
When merged, I will tag it as 2.1.8.**1**

File: README.md (hunk 1 above)
Comment (robertbeal) [reply README.md]: Ah cheers!

Comment (tomsquest) [reply README.md]: Can you fix it ?

Comment (robertbeal) [reply README.md]: Oh heh, sorry, wasn't looking and thought you'd corrected it. I'll do it now!

Comment (robertbeal) [reply README.md]: done!

**Pull request merged**